src/app/app.module.ts

```diff
@@ -1,18 +1,18 @@
 import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { BrowserModule } from '@angular/platform-browser';
 import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { CourseComponent } from './features/course/course.component';
 import { CoursesComponent } from './features/courses/courses.component';
-import { CoursesModule } from './features/courses/courses.module';
 import { LoginComponent } from './features/login/login.component';
 import { RegistrationComponent } from './features/registration/registration.component';
 import { HeaderComponent } from './shared/components/header/header.component';
+import { CourseCrudComponent } from './features/course/course-crud/course-crud.component';
 
 @NgModule({
-    declarations: [AppComponent, LoginComponent, RegistrationComponent, CourseComponent, CoursesComponent,HeaderComponent],
-    imports: [BrowserModule, AppRoutingModule,CoursesModule, FontAwesomeModule],
+    declarations: [AppComponent, LoginComponent, RegistrationComponent, CoursesComponent,HeaderComponent, CourseCrudComponent],
+    imports: [BrowserModule, AppRoutingModule, FontAwesomeModule,FormsModule,ReactiveFormsModule],
```

Comment:
FontAwesomeModule,FormsModule,ReactiveFormsModule should be moved to shared module